visual-js/visual-snapshots/src/commands/params.ts

```diff
@@ -0,0 +1,19 @@
+export const getParam = (
+  options: { [key: string]: string },
+  optionName: string,
+  envVariableName: string,
+  defaultValue: string | undefined = undefined,
+) => {
+  if (options[optionName]) {
+    return options[optionName];
+  }
+  if (process.env[envVariableName]) {
+    return process.env[envVariableName];
+  }
+  if (defaultValue) {
+    return defaultValue;
+  }
+  throw new Error(
+    `Missing required argument. Please provide ${optionName} option, or set ${envVariableName} environment variable.`,
+  );
+};
```

Comment:
Do you intend to use this for getting options from either the command options passed from CLI or env variables? If so, maybe a better option would be to pass defaults into the `Option`s themselves?

https://github.com/tj/commander.js?tab=readme-ov-file#default-option-value

Reply:
We need to get the option value, or env variable value, or default value.
I could use the default and set to to env variable value or the default value... but not every option has a default value.
In my opinion, this is clearer this way and in case the value is not determined but required we throw an exception in a single place